### PR TITLE
ZTE-SH-CN-run-application-run-stateless-application-deployment-pr-2017-09-08

### PR DESCRIPTION
**Pull request by jianglingxia**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5348)

### CODE REVIEW

File: cn/docs/tasks/run-application/run-stateless-application-deployment.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+---
+
+{% capture overview %}
+
+本文介绍通过Kubernetes Deployment对象如何去运行一个应用.
+
+{% endcapture %}
+
+
+{% capture 目标 %}
```
Comment (chenopis) [inline cn/docs/tasks/run-application/run-stateless-application-deployment.md]: The `capture` variable names need to be in English and follow the ones outlined in https://kubernetes.io/docs/home/contribute/page-templates/#task_template, i.e.:

* `overview`
* `prerequisites`
* `steps`
* `discussion`
* `whatsnext`

**Comment by chenopis** [comment]
cc @markthink

File: cn/docs/tasks/run-application/run-stateless-application-deployment.md (hunk 1 above)
Comment (jianglingxia) [reply cn/docs/tasks/run-application/run-stateless-application-deployment.md]: ok ,thanks,I have updated it!

File: cn/docs/tasks/run-application/run-stateless-application-deployment.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+## 删除deployment
+
+通过名称删除deployment:
+
+    kubectl delete deployment nginx-deployment
+
+## ReplicationControllers -- 旧的方式
+
+创建一个replicated application首选方法是使用Deployment,
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/tasks/run-application/run-stateless-application-deployment.md]: 我觉得 多副本的应用。这种意译更恰当。

Comment (zhangxiaoyu-zidif) [reply cn/docs/tasks/run-application/run-stateless-application-deployment.md]: @markthink do you think so?

Comment (warmchang) [reply cn/docs/tasks/run-application/run-stateless-application-deployment.md]: I agree with you. "replicated application" --> "多副本应用".

Comment (jianglingxia) [reply cn/docs/tasks/run-application/run-stateless-application-deployment.md]: thanks for you @warmchang @zhangxiaoyu-zidif 
I have updated it!

**Comment by markthink** [comment]
@jianglingxia Please close PR, update to k8smeetup library, unified submission..Do not submit a single file..

update table:
https://docs.google.com/spreadsheets/d/1FDFCv9RK5nSMgLXhPrJ5k7r5QvHnNEFnXbvoFiM8v20/edit#gid=1015862274

**Pull request closed without merging**